packages/nextjs/components/ExampleUi/ContractInteraction/UiDescription.tsx

```diff
@@ -4,19 +4,19 @@ import { XMarkIcon } from "@heroicons/react/24/outline";
 export const UiDescription = () => {
   const [visible, setVisible] = useState(true);
   return (
-    <div className={`mt-7 flex gap-2 ${visible ? "" : "invisible"} max-w-2xl`}>
+    <div className={`mt-10 flex gap-2 ${visible ? "" : "invisible"} max-w-2xl`}>
       <div className="flex gap-5 bg-base-200 bg-opacity-80 z-0 p-7 rounded-2xl shadow-lg">
         <span className="text-3xl">👋🏻</span>
         <div>
-          <span className="block">
+          <div>
             In this page you can see how some of our <strong>hooks & components</strong> work, and how you can bring
             them to life with your own design! Have fun and try it out!
-          </span>
-          <span className="block mt-2">
+          </div>
+          <div className="mt-2">
             Check out{" "}
-            <code className="italic bg-base-300 text-base font-bold">packages/nextjs/pages/example-ui.tsx</code> and its
```

Comment:
I think I prefer this without the spaces :D

Reply:
it breaks styles on mobiles, word-break solutions looking bad :/

Reply:
A negative `word-spacing` fixes this.

Reply:
cool idea, thanks! 🎉 
updated